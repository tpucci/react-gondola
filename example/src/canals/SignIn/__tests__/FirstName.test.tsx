import React from 'react';
import TestRenderer from 'react-test-renderer';

import { FirstName } from '../FirstName';

describe('FirstName', () => {
  it('renders correctly', () => {
    const testRenderer = TestRenderer.create(<FirstName />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
