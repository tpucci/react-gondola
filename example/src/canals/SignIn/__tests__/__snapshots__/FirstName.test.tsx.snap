// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstName renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFAAAA",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    FirstName
  </Text>
</View>
`;
